src/python/visclaw/data.py

```diff
@@ -621,12 +622,17 @@ def __init__(self, name, figno, fig_type, plotdata):
         self._show = True
         self.add_attribute('plotaxes_dict', {})
         self.add_attribute('type',fig_type)   # = 'each_frame' or 'each_run' or 'each_gauge'
+        self.add_attribute('use_for_kml',False)
+        self.add_attribute('kml_dpi',200)
+        self.add_attribute('kml_xlimits',None)
```

Comment:
Sure - that is what I do now if #kml_xlimits/ylimits" are not set.  

The question then, is, which axes would we grab the xlimts/ylimits from, and how does this get communicated back to plotpages (where the kml files are created).    It seems to make sense to grab the axes information from the type "2d_color" plot. 

Ultimately, what we'd want to avoid is a situation where the plots that are generated don't line up exactly with the GE topography, but that the user is left with no obvious reason why.  Since they already have to set several things in the plotfigure attributes, one more item (x/y limits) didn't seem like such a big deal.